labelparse.py

```diff
@@ -37,19 +33,19 @@ def repo_fetch():
 
     for repo in dtk_repo_names:
         response = requests.get(
-            'https://api.github.com/repos/newrelic/' + str(repo) + '/issues')
-        if response.status_code == 200:
+            f'https://api.github.com/repos/newrelic/{repo}/issues'
+        )
+        if response:
             for issue in response.json():
                 print_issue(issue)
-                for label in issue['labels']:
-                    if label['name'] in labels_count:
-                        labels_count[label['name']] += 1
-                    else:
-                        labels_count[label['name']] = 1
+                for label in issue.get('labels', []):
+                    labels_count[label['name']] = (
+                        labels_count.setdefault(label['name'], 0) + 1
```

Comment:
Any `dict` value sets where you have the form:

```
if k in d:
    d[k] += 1
else:
    d[k] = 1
```

You can simplify with:

```
d[k] = d.setdefault(k, 0) + 1
```

This is a little cleaner and makes use of the nice `setdefault()` function. `setdefault()` returns the existing value, if applicable. If there is no value there, it sets the value you pass as the default and then returns the existing value.

So the first iteration, it will set to (0 + 1), then the second, it'll set to (1 + 1), and so on.